scipy/optimize/_lbfgsb_py.py

```diff
@@ -171,6 +171,44 @@ def fmin_l_bfgs_b(func, x0, fprime=None, args=(),
       FORTRAN routines for large scale bound constrained optimization (2011),
       ACM Transactions on Mathematical Software, 38, 1.
 
+    Examples
+    --------
+    Solve a linear regression problem via `fmin_l_bfgs_b`. To do this, first we define an objective function ``f(m,b) = 
+    (y - y_model)**2``, where `y` describes the observations and `y_model` the prediction of the linear model as 
+    ``y_model = m * x + b``. The bounds for the parameters, m and b, are arbitrarily chosen as (0,5) and (5,10) 
+    for this example.
+
+    >>> import numpy as np
+    >>> from scipy.optimize import fmin_l_bfgs_b
+    >>> X = np.arange(0, 10, 1)
+    >>> M = 2
+    >>> B = 3
+    >>> Y = M * X + B
+    >>> def func(parameters, *args):
+    ...     x = args[0]
+    ...     y = args[1]
+    ...     m, b = parameters
+    ...     y_model = m * x+b
```

Comment:
```suggestion
    ...     y_model = m*x + b
```

Reply:
@dschmitz89 I see this was your suggestion above, am I wrong here? Does `+` bind more strongly?

Reply:
@lucascolley missed that, thanks!